modules/manage/pages/manage-backup-and-restore/manage-backup-and-restore.adoc

```diff
@@ -0,0 +1,799 @@
+
+To pause a backup, access the *Repositories* screen, and left-click on the row for the repository to be paused.
+This expands the row vertically, and displays the *Pause* button.
+Left-click on this, to pause backups:
+
+image:manage-backup-restore/pauseButton.png[,240,align=left]
+
+The button now changes into a *Resume* button.
+Left-click on this whenever backups are to be resumed:
+
+image:manage-backup-restore/resumeButton.png[,240,align=left]
+
+[#archive-repositories]
+== Archive Repositories
+
+When a repository should no longer receive fresh backups, the repository can be _archived_.
+This means that it remains available, with all its data; but no longer receives modifications.
+To archive a repository, proceed as follows;
+
+. Access the repository that is to be archived on the *Repositories* screen, and expand the row for the repository by left-clicking on the repository's row.
+When the row has expanded, left-click on the *Archive* button:
++
+image:manage-backup-restore/archiveButton.png[,90,align=left]
++
+This brings up the *Archive Repository* dialog, which appears as follows:
++
+image:manage-backup-restore/archiveRepositoryDialog.png[,420,align=left]
++
+The dialog contains a notification, warning that no further backups or merges will be possible to the repository, once it has been archived.
+
+. Confirm the repository to be archived.
+Enter its name into the *Confirm repository ID to archive* field.
+Then, enter the intended name of the _archived_ version of the repository into the *New ID* field.
+The dialog now appears as follows:
++
+image:manage-backup-restore/archiveRepositoryDialogComplete.png[,420,align=left]
++
+Left-click on *Archive*, to confirm.
+The dialog now disappears.
+
+. Check the archived repository, on the *Repositories* screen, in the *Archived Repositories* panel.
+This now appears as follows:
++
+image:manage-backup-restore/archivedRepository.png[,720,align=left]
++
+Left-click on the row for the archived repository, to expand the row, and reveal details and options:
++
+image:manage-backup-restore/archivedRepositoryExpandedRow.png[,720,align=left]
++
+The backups within and tasks used for the repository can be examined by means of the *Inspect Backups* and *Tasks* buttons.
+The *Restore* button brings up the same dialog as described above, in xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#restore-backups[Restore Backups].
+The *Delete* button is described immediately below, in xref:manage:manage-backup-and-restore/manage-backup-and-restore.adoc#delete-repositories[Delete Repositories].
+
+[#delete-repositories]
+== Delete Repositories
+
+Backups cannot be individually deleted: they can be deleted only by repository.
```

Comment:
Backups can be deleted individauly from the "data/inspect backups page" you willnotcie that each row has a trash icon at the end, clicking that icon will allow the user to delete a backup. This action is unreversable, once a backup is delted there is no way to bring it back.

Reply:
I've removed the statement about backups cannot be individually deleted. And, in the Inspect Backups section, I've included a subsection named Delete Backups, which shows how to use the icon and the confirmation dialog.  At https://github.com/couchbase/docs-server/pull/1596